generator/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "require-dev": {
+        "php" : "^7",
+        "phpunit/phpunit": "^7.4",
+        "smarty/smarty": "^3.1"
```

Comment:
Smarty should be under the `require` key

Reply:
☑️

Reply:
☑️